catalog/to-consul/resource_test.go

```diff
@@ -1714,98 +999,17 @@ func TestServiceResource_clusterIPPrefix(t *testing.T) {
 	actual := syncer.Registrations
 	require.Len(actual, 2)
 	require.Equal("prefixfoo", actual[0].Service.Service)
-	require.Equal("1.2.3.4", actual[0].Service.Address)
+	require.Equal("1.1.1.1", actual[0].Service.Address)
 	require.Equal(8080, actual[0].Service.Port)
 	require.Equal("prefixfoo", actual[1].Service.Service)
-	require.Equal("2.3.4.5", actual[1].Service.Address)
-	require.Equal(8080, actual[1].Service.Port)
-	require.NotEqual(actual[0].Service.ID, actual[1].Service.ID)
-}
-
-// Test that the proper registrations are generated for a ClusterIP type with multiple ports.
-func TestServiceResource_clusterIPMultiEndpoint(t *testing.T) {
```

Comment:
which test did this dupe?

Reply:
This test became redundant because `clusterIPService` and `createEndpoints` functions create multiple ports by default, and so this functionality is now tested in `TestServiceResource_clusterIP`.